src/components/titlePanel/createTitlePanel.ts

```diff
@@ -68,13 +79,30 @@ const createTitlePanel: TitlePanelFactory = createWidgetBase
 					};
 				}
 
+				titleProperties.role = 'heading';
+
+				if (ariaHeadingLevel) {
+					titleProperties['aria-level'] = String(ariaHeadingLevel);
+				}
+
+				if (open) {
+					// FIXME - id of content node
+					titleButtonProperties['aria-controls'] = 'content';
```

Comment:
This seems very awkward to me since with vdom, in the closed state there is no content pane existing in the DOM. So we set `aria-controls="<id of non-existent element>"`?